wp-config-docker.php

```diff
@@ -0,0 +1,136 @@
+<?php
+/**
+ * The base configuration for WordPress
+ *
+ * The wp-config.php creation script uses this file during the
+ * installation. You don't have to use the web site, you can
+ * copy this file to "wp-config.php" and fill in the values.
+ *
+ * This file contains the following configurations:
+ *
+ * * MySQL settings
+ * * Secret keys
+ * * Database table prefix
+ * * ABSPATH
+ *
+ * @link https://codex.wordpress.org/Editing_wp-config.php
+ *
+ * @package WordPress
+ */
+
+/**
+ * Set up our global environment constant and load its config first
+ * Default: production
+ */
+define('WP_ENV', getenv('WP_ENV') ?: 'production');
+
+// ** MySQL settings - You can get this info from your web host ** //
+/** The name of the database for WordPress */
+define( 'DB_NAME', getenv('DB_NAME') ?: 'wordpress' );
+
+/** MySQL database username */
+define( 'DB_USER', getenv('DB_USER') ?: '' );
+
+/** MySQL database password */
+define( 'DB_PASSWORD', getenv('DB_PASSWORD') ?: '' );
+
+/** MySQL hostname */
+define( 'DB_HOST', getenv('DB_HOST') ?: 'localhost' );
+
+/** Database Charset to use in creating database tables. */
+define( 'DB_CHARSET', getenv( 'DB_CHARSET' ) ?: 'utf8' );
+
+/** The Database Collate type. Don't change this if in doubt. */
+define( 'DB_COLLATE', getenv( 'DB_COLLATE' ) ?: '' );
+
+/**#@+
+ * Authentication Unique Keys and Salts.
+ *
+ * Change these to different unique phrases!
+ * You can generate these using the {@link https://api.wordpress.org/secret-key/1.1/salt/ WordPress.org secret-key service}
+ * You can change these at any point in time to invalidate all existing cookies. This will force all users to have to log in again.
+ *
+ * @since 2.6.0
+ */
+define( 'AUTH_KEY',         getenv( 'AUTH_KEY' ) ?: 'put your unique phrase here' );
+define( 'SECURE_AUTH_KEY',  getenv( 'SECURE_AUTH_KEY' ) ?: 'put your unique phrase here' );
+define( 'LOGGED_IN_KEY',    getenv( 'LOGGED_IN_KEY' ) ?: 'put your unique phrase here' );
+define( 'NONCE_KEY',        getenv( 'NONCE_KEY' ) ?: 'put your unique phrase here' );
+define( 'AUTH_SALT',        getenv( 'AUTH_SALT' ) ?: 'put your unique phrase here' );
+define( 'SECURE_AUTH_SALT', getenv( 'SECURE_AUTH_SALT' ) ?: 'put your unique phrase here' );
+define( 'LOGGED_IN_SALT',   getenv( 'LOGGED_IN_SALT' ) ?: 'put your unique phrase here' );
+define( 'NONCE_SALT',       getenv( 'NONCE_SALT' ) ?: 'put your unique phrase here' );
+
+/**#@-*/
+
```

Comment:
We need a new category for error reporting. Set by default to output all errors.

```suggestion
/**
 * Error reporting
 **/
 error_reporting(E_ALL);
```